test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5257,7 +5277,7 @@ def save_configuration(self, outfile, mode='a'):
         :returns: True on success, False on error
         """
         conf = {}
-        sconf = {MGR_OBJ_SERVER: conf}
+        self.saved_config[MGR_OBJ_SERVER] = {}
```

Comment:
I think this line is not necessary as after the configuration is stored, you are initializing `self.saved_config[MGR_OBJ_SERVER]`. Same for `MGR_OBJ_SCHED`.